examples/message/labview/LabVIEW.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using NationalInstruments.SystemLink.Clients.Message;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Message
+{
+    /// <summary>
+    /// Example for the SystemLink Message Client API that communicates with
+    /// the Host.vi from the Async Messaging LabVIEW example.
+    /// </summary>
+    class LabVIEW
+    {
+        /// <summary>
+        /// The delay between sending periodic messages to the host topic.
+        /// </summary>
+        static readonly TimeSpan MessageDelay = TimeSpan.FromSeconds(1);
+
+        /// <summary>
+        /// The message topic for sending messages to the Host VI.
+        /// </summary>
+        const string HostTopic = "Example to Host";
```

Comment:
Don't we conventionally use PascalCase for message topics? Although if the VI uses this format and is already shipping, there isn't much we can do to change it...

Reply:
Yes, this is what the LabVIEW example is already using. We don't really put many restrictions on topic names.